lib/components/Blocks/Header/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { FlexBox } from "@/components/Layout/_FlexBox"
+import { Avatar, AvatarFallback, AvatarImage } from "@/ui/avatar"
+import { forwardRef } from "react"
+
+interface HeaderProps {
+  title: string
+  subtitle: string
+  src: string
+  alt: string
+}
+
+export const Header = forwardRef<HTMLDivElement, HeaderProps>(
+  ({ title, subtitle, src, alt }, ref) => {
+    return (
+      <FlexBox ref={ref} className="px-10 py-6">
```

Comment:
I'm personally not sold yet on using `Stack`, `Split` and `FlexBox` internally. I think they are nice components as an escape hatch for the consumers, but maybe using tailwind classes here would work better? I can see how some components like `Stack` will generate friction when we need to make things responsive and react to container queries.

Reply:
Makes sense, I assumed they were there to be reused, but I don't see the upside either.